scenarios/throughput_stress.go

```diff
@@ -38,12 +41,33 @@ type tpsExecutor struct {
 }
 
 func (t *tpsExecutor) Run(ctx context.Context, info loadgen.ScenarioInfo) error {
+	// Parse scenario options
+	internalIterations := info.ScenarioOptionInt(IterFlag, 5)
+	internalIterTimeout := info.ScenarioOptionDuration(IterTimeout, time.Minute)
+	continueAsNewCount := info.ScenarioOptionInt(CANEventFlag, 120)
+	workflowIDPrefix := cmp.Or(info.ScenarioOptions[WorkflowIDPrefix], defaultWorkflowIDPrefix)
+	nexusEndpoint := info.ScenarioOptions[NexusEndpointFlag] // disabled by default
+	skipSleep := info.ScenarioOptionBool(SkipSleepFlag, false)
+	sleepActivityPriorityDistribution, err := loadgen.ParseIntDistribution(info.ScenarioOptions[SleepActivityPriorityDistributionFlag])
+	if err != nil {
+		return fmt.Errorf("failed to parse %v: %w", SleepActivityPriorityDistributionFlag, err)
+	}
+	sleepActivityDurationDistribution, err := loadgen.ParseDurationDistribution(info.ScenarioOptions[SleepActivityDurationDistributionFlag])
+	if err != nil {
+		return fmt.Errorf("failed to parse %v: %w", SleepActivityDurationDistributionFlag, err)
+	}
+	visibilityVerificationTimeout, err := time.ParseDuration(cmp.Or(info.ScenarioOptions[VisibilityVerificationTimeout], "3m"))
+	if err != nil {
+		return fmt.Errorf("failed to parse %v: %w", VisibilityVerificationTimeout, err)
+	}
+	timeout := time.Duration(1*internalIterations) * internalIterTimeout
```

Comment:
Moved all of these to the top so (a) if there's an error, it aborts immediately and (b) so it's not re-parsed on every run.